src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.Properties.cs

```diff
@@ -0,0 +1,477 @@
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Navigation;
+using Microsoft.UI.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Windows.UI.Input;
+using Windows.Devices.Input;
+#endif
+
+namespace Uno.Toolkit.UI;
+
+public partial class ZoomContentControl
+{
+	#region DependencyProperty: HorizontalOffset
+
+	/// Identifies the HorizontalOffset dependency property.
+	public static DependencyProperty HorizontalOffsetProperty { get; } = DependencyProperty.Register(
+		nameof(HorizontalOffset),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double), OnHorizontalOffsetChanged));
+
+	/// Gets or sets the horizontal offset for panning the content.
+	public double HorizontalOffset
+	{
+		get => (double)GetValue(HorizontalOffsetProperty);
+		set => SetValue(HorizontalOffsetProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: HorizontalScrollValue
+
+	/// Identifies the HorizontalScrollValue dependency property.
+	public static DependencyProperty HorizontalScrollValueProperty { get; } = DependencyProperty.Register(
+		nameof(HorizontalScrollValue),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double)));
+
+	/// Gets or sets the value of the horizontal scrollbar.
+	/// It's used to represent the scroll position within the scroll bar UI
+	public double HorizontalScrollValue
+	{
+		get => (double)GetValue(HorizontalScrollValueProperty);
+		set => SetValue(HorizontalScrollValueProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: HorizontalMinScroll
+
+	/// Identifies the HorizontalMinScroll dependency property.
+	public static DependencyProperty HorizontalMinScrollProperty { get; } = DependencyProperty.Register(
+		nameof(HorizontalMinScroll),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double)));
+
+	/// Gets or sets the minimum horizontal scroll limit.
+	public double HorizontalMinScroll
+	{
+		get => (double)GetValue(HorizontalMinScrollProperty);
+		set => SetValue(HorizontalMinScrollProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: HorizontalMaxScroll
+
+	/// Identifies the HorizontalMaxScroll dependency property.
+	public static DependencyProperty HorizontalMaxScrollProperty { get; } = DependencyProperty.Register(
+		nameof(HorizontalMaxScroll),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(default(double)));
+
+	/// Gets or sets the maximum horizontal scroll limit.
+	public double HorizontalMaxScroll
+	{
+		get => (double)GetValue(HorizontalMaxScrollProperty);
+		set => SetValue(HorizontalMaxScrollProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: HorizontalZoomCenter
+
+	/// Identifies the HorizontalZoomCenter dependency property.
```

Comment:
I haven't really tried the control in practice, but if zooming with two fingers on a mobile device, is this property used? It feels like the zoom center should be where the first finger touched the screen?